lib/Component/VCard.php

```diff
@@ -411,6 +411,24 @@ function preferred($propertyName) {
     }
 
     /**
+     * Returns a specific type of multi fields (ADR, TEL, or EMAIL).
+     *
+     * @param string $propertyName
+     * @param string $type
+     *
+     * @return VObject\Property|null
+     */
+    function getForType($propertyName, $type) {
```

Comment:
I would suggest renaming this to `getByType` to be more in line with other functions.